config/src/config/indexer_table_info_config.rs

```diff
@@ -4,8 +4,9 @@
 use serde::{Deserialize, Serialize};
 
 // Useful defaults
-pub const DEFAULT_PARSER_TASK_COUNT: u16 = 20;
-pub const DEFAULT_PARSER_BATCH_SIZE: u16 = 1000;
+pub const DEFAULT_PARSER_TASK_COUNT: u16 = 10;
+pub const DEFAULT_PARSER_BATCH_SIZE: u16 = 100;
```

Comment:
I think that we talked about not changing these without proper load testing?

Reply:
lol i see that in the next PR you reverted it back. Let's not make the change in the first place lol